### PR TITLE
Update G suite Auditor README.md

### PR DESCRIPTION
**Pull request by ostolero**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Related to this [thread](https://panw-global.slack.com/archives/C011E5T5K17/p1653466615210959), a customer couldn't find the integrations permissions.
adding them to the readme and not only the description.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 G Suite Auditor is an integration that receives audit logs from G Suite's different applications - admin, drive, calendar, and more.
 
+## Required Permissions:
+In order to use the integration you will need:
+* Service Account key.
+* Admin account.
+* Permission to the following scope - https://www.googleapis.com/auth/admin.reports.audit.readonly
+
+For more information see the integrations description.
```
Comment (JudahSchwartz) [inline Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md:9]:
```suggestion
For more information, see the integrations description.
```

File: Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 G Suite Auditor is an integration that receives audit logs from G Suite's different applications - admin, drive, calendar, and more.
 
+## Required Permissions:
+In order to use the integration you will need:
+* Admin account.
+* Permission to the following scope - https://www.googleapis.com/auth/admin.reports.audit.readonly .
+
+For more information, see the integrations description.
```
Comment (ShahafBenYakir) [inline Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md:8]: Usually, we are saying these exact words in the integration description (For more information, see the integrations documentation) as it makes more sense to have more information on the integration docs than description. What is there which is not here?

File: Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 G Suite Auditor is an integration that receives audit logs from G Suite's different applications - admin, drive, calendar, and more.
 
+## Required Permissions:
+In order to use the integration you will need:
+* Admin account.
```
Comment (ShahafBenYakir) [inline Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md:5]: Are you sure of this? Normal users cannot interact with the API only admins can?

Comment (ostolero) [reply Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md:5]: It looks like you need the Admin email address. No password is required

File: Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md (hunk 2 above)
Comment (ostolero) [reply Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md:8]: It is the same information. As a client couldn't find the permission section I added this.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**